### PR TITLE
fix(70601): fix qty selected package  per group --run-build

### PR DESCRIPTION
**Pull request by imam-auliya**
> [#70601] - FE POS - Done

Acceptance Criteria
- [x] qty item selected menyesuaikan

Changelog:  
- #70601 create method to calculate item selected per group

image

### CODE REVIEW

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=75) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=75)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=75)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=75)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=75)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=75&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=75)

File: src/app/mocks/menuitem.ts [hunk 1]
```diff
@@ -78,6 +78,7 @@ export interface MenuPackage {
 }
 
 export interface MenuPackageGroup {
+  menu_package: any
```
Comment (aldiz1711) [inline src/app/mocks/menuitem.ts:81]: jangan pakai any

**Review by aldiz1711 — CHANGES_REQUESTED** [review]
ada beberapa comment tolong di cek dulu ya

File: src/app/mocks/menuitem.ts (hunk 1 above)
Comment (imam-auliya) [reply src/app/mocks/menuitem.ts]: Ok

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=75) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=75)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=75)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=75)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=75)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=75&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications: **Duplication**

**Pull request merged**